content/4_Gold/Modular.mdx

```diff
@@ -81,6 +81,14 @@ multiply as normal.
 	>
 		some practice probs
 	</Resource>
+    <Resource
+        source="MONT@Modern Olympiad Number Theory"
+        title="Chapters 2 & 5 - Basic to Advanced Modular Arithmetic"
+        url="https://drive.google.com/file/d/1BcJTLjQaelZ4w_70oHKyImC2I8zLfyrt/view"
+        starred
+    >
+        Really good for beginners to advanced
```

Comment:
I don't think you need this line. Starring it should be sufficient

Reply:
Ok I did that